style.css

```diff
@@ -1,27 +1,33 @@
 * {
 	box-sizing: border-box;
 }
-h1{
-    font-size:auto;
+html {
+	font-family: 'Poppins', sans-serif;
+	font-size:10px;
+
 }
+
 body {
 	background-color: #F7F8F8;
+	padding-bottom: 5rem;
+	font-size:1.6rem;
```

Comment:
Space after colon or no space? Pick one and run with it. Tip: we use space after colon at YW.

Reply:
Gone through and changed all spacing to space after colon